### ISSUE TITLE
ext_smtp.py TLS without SSL

### ISSUE THREAD

**Issue by TomFreudenberg** [datafolklabs/cement#667]
The activation for TLS is only possible on SSL connection.

That is not necessary nor that all services have enabled SSL.

https://github.com/datafolklabs/cement/blob/ea320570d7951483955637cfd8370bd7744f723e/cement/ext/ext_smtp.py#L112-L114

The TLS starttls command should be possible for non-SSL connections as well to make sure that the message channel is encrypted.

Would you like me to send in a PR for that?

Best regards
Tom

**Comment by TomFreudenberg** [comment]
The code

https://github.com/datafolklabs/cement/blob/ea320570d7951483955637cfd8370bd7744f723e/cement/ext/ext_smtp.py#L108-L119

should be:

```python
        if is_true(params['ssl']):
            server = smtplib.SMTP_SSL(params['host'], params['port'],
                                      params['timeout'])
            LOG.debug("%s : initiating ssl" % self._meta.label)

        else:
            server = smtplib.SMTP(params['host'], params['port'],
                                  params['timeout'])

        if is_true(params['tls']):
            LOG.debug("%s : initiating tls" % self._meta.label)
            server.starttls()

```

**Comment by derks** [comment]
Resolved in `main`, will be released with `stable/3.0.10`.
